src/components/MapView/DateSelector/index.tsx

```diff
@@ -1,6 +1,8 @@
 import React, { forwardRef, Ref, useEffect, useRef, useState } from 'react';
+import { useTranslation } from 'react-i18next';
 import { useSelector } from 'react-redux';
 import Moment from 'moment';
+import 'moment/locale/km';
```

Comment:
This was loaded here as an example. We probably want to load this at a higher level, likely in `i18n.ts` ?